api-test/src/test/java/org/openmrs/module/metadatasharing/handler/impl/ConceptReferenceTermHandlerTest.java

```diff
@@ -0,0 +1,70 @@
+package org.openmrs.module.metadatasharing.handler.impl;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.UUID;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.api.ConceptService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.metadatasharing.MetadataSharing;
+import org.openmrs.module.metadatasharing.SerializedPackage;
+import org.openmrs.module.metadatasharing.wrapper.PackageImporter;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+
+public class ConceptReferenceTermHandlerTest extends BaseModuleContextSensitiveTest {
+	
+	private ConceptService conceptService;
+	
+	public static final String CONCEPT_SOURCE_UUID = UUID.randomUUID().toString();
+	
+	public static final String CONCEPT_REFERENCE_TERMS_HEADER = "mds-concept-reference-terms-header.xml";
+	
+	public static final String CONCEPT_REFERENCE_TERMS_METADATA = "mds-concept-reference-terms-metadata.xml";
+	
+	private String headerXml;
+	

```

Comment:
Do you have any reason as to why these are class level instead of local variables?